cmd/storage.go

```diff
@@ -49,19 +51,28 @@ var storageDeleteCmd = &cobra.Command{
 		if len(args) != 0 {
 			storageName = args[0]
 		}
+		var confirmDeletion string
 		applicationName, err := application.GetCurrent(client)
 		checkError(err, "")
 		projectName := project.GetCurrent(client)
 		componentName := getComponent(client, storageComponent, applicationName, projectName)
-
-		err = storage.Remove(client, storageName, applicationName, componentName)
-		checkError(err, "failed to delete storage")
-
-		switch storageName {
-		case "":
-			fmt.Printf("Deleted all storage from %v\n", componentName)
-		default:
-			fmt.Printf("Deleted %v from %v\n", storageName, componentName)
+		if storageForceDeleteflag {
+			confirmDeletion = "y"
+		} else {
+			fmt.Printf("Are you sure you want to delete the storage: %v? [y/N] ", storageName)
+			fmt.Scanln(&confirmDeletion)
+		}
+		if strings.ToLower(confirmDeletion) == "y" {
+			err = storage.Remove(client, storageName, applicationName, componentName)
+			checkError(err, "failed to delete storage")
+			switch storageName {
+			case "":
```

Comment:
deleting all storage from component was removed in https://github.com/redhat-developer/odo/pull/463

This should be addressed in rebase